runtime/compiler/p/codegen/J9PPCSnippet.cpp

```diff
@@ -1846,15 +1846,12 @@ uint32_t TR::getCCPreLoadedCodeSize()
       size += 4 + (linesToLdPrefetch + 1) / 2 * 4 + 1 + (l3SkipLines ? 2 : 0) + (linesToStPrefetch + 1) / 2 * 4;
       }
    size += 3;
-
-      //TR_ObjectAlloc/TR_ConstLenArrayAlloc/TR_VariableLenArrayAlloc/Hybrid arraylets
-      size += (2 + (11 + 11) + 2) + (16 + 14) + 4 + 9 + 4;
-      // If TLH prefetching is done
-      size += 2*5 + 2*5;
-   #if defined(DEBUG)
-      //TR_ObjectAlloc/TR_ConstLenArrayAlloc
-      size += 5 + 8;
-   #endif
+   // If TLH prefetching is done
+   size += 2*5 + 2*5;
+#if defined(DEBUG)
+   //TR_ObjectAlloc/TR_ConstLenArrayAlloc
+   size += 5 + 8;
+#endif
```

Comment:
This code should also be removed, as this is all related to calculating the size of the removed snippets.